components/UI/profileCard/profileCard.tsx

```
@@ -122,6 +124,21 @@ const ProfileCard: FunctionComponent<ProfileCardProps> = ({
         </div>
 
         <div className="flex flex-col h-full justify-center">
+          {totalBalance === null ? ( // Check if balance is still loading
+            <>
+            //Updated the Specific Profile Card Component with skeleton Loading component
```

Biome

[error] Wrap comments inside children within braces.

Unsafe fix: Wrap the comments with braces


(lint/suspicious/noCommentText)